packages/oscal-react-library/src/components/OSCALCatalogGroup.tsx

```diff
@@ -112,6 +89,13 @@ interface CollapsibleListItemProps extends CatalogGroupFragmentProps {
    * Callback function to set list item opened
    */
   readonly setListItemOpened: (...args: any[]) => void;
+
+  /**
```

Comment:
Not sure if this space was intentional or not
```suggestion
  readonly setListItemOpened: (...args: any[]) => void;
  /**
```

Reply:
Absolutely intention.